maths/gamma.py

```diff
@@ -0,0 +1,32 @@
+from math import factorial
+
+def gamma(num):
+    '''
+    >>> gamma(1)
+    1
+
+    >>> gamma(0)
+    Traceback (most recent call last):
+        ...
+    ValueError:  math domain error
+    
+    >>> gamma(10)
+    362880
+    
+    >>> gamma(-199)
+    Traceback (most recent call last):
+        ...
+    ValueError:  math domain error
+    '''
```

Comment:
Can we test these results against https://docs.python.org/3/library/math.html#math.gamma especially with floating point and negative numbers.

Reply:
Added . The function had to be re-written as the old version would only work with positives integers greater than one.